packages/pwa-kit-create-app/assets/pwa/default.js

```diff
@@ -9,6 +9,29 @@ exports.template = ({commerceApi, einsteinApi}) => `module.exports = {
         url: {
             locale: 'none'
         },
+        defaultSite: 'RefArch',
+        sites: [
+            {
+                id: 'RefArch',
+                l10n: {
+                    supportedCurrencies: ['USD'],
+                    defaultCurrency: 'USD',
+                    defaultLocale: 'en-US',
+                    supportedLocales: [
+                        {
+                            id: 'en-US',
+                            alias: 'en',
```

Comment:
Should we have these defined if we aren't using them?

Reply:
ah good catch, we are not currently using it, I will remove it in the next commit